app/controllers/v1/recipes_controller.rb

```diff
@@ -1,6 +1,17 @@
 # frozen_string_literal: true
 
 class V1::RecipesController < ApplicationController
+
+  def index
+    recipes = Recipe.all
+
+    if recipes.empty?
+      render json: { message: 'There are no recipes here' }
+    else
```

Comment:
I will again suggest that you just render an empty collection and deal with it on the client-side. The way you do it now will still require you to add an error handler when you receive the response.